pkg/api/types.go

```diff
@@ -131,6 +131,9 @@ type PodSpec struct {
 	// PodSpot is the policy that determines if a spot instance may be used for
 	// a Pod.
 	Spot PodSpot `json:"spot,omitempty"`
+	// PodDedicated is a policy that determines if a dedicated host instance may
+	// be used for a pod
```

Comment:
Can you pls update the comment to match the name/type of the field?

Reply:
Reasoning for this was because i was originally going to set up dedicated support like we have spot support with policies e.g `SpotAlways`, and `SpotNever`. However I dont think this is necessary and we can instead just use boolean. 

I have updated the comment.